src/routes/index.js

```diff
@@ -90,4 +93,14 @@ export default (zoapp) => {
     ["admin", "master", "application"],
     admin.getParameterValue,
   );
+
+  console.log(config.backend.managementEndpoint)
```

Comment:
If this log is made to be kept its must use logger.info IMO

Reply:
I'll remove that.